backend/src/sockets/handler.ts

```diff
@@ -7,6 +7,11 @@ import Video from "../models/video";
 import uniqid from "uniqid";
 import Database from "../core/database";
 
+interface Message {
```

Comment:
Maybe move this to an external file in the `models` folder. I anticipate we will be moving this `models` folder to a shared module